sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/models/PurchasedPhoneNumber.java

```diff
@@ -4,60 +4,60 @@
 
 package com.azure.communication.phonenumbers.models;
 
-import com.azure.core.annotation.Fluent;
+import com.azure.core.annotation.Immutable;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.time.OffsetDateTime;
 
 /** The PurchasedPhoneNumber model. */
-@Fluent
+@Immutable
 public final class PurchasedPhoneNumber {
     /*
      * The id of the phone number, e.g. 11234567890.
      */
-    @JsonProperty(value = "id", required = true)
+    @JsonProperty(value = "id", required = true, access = JsonProperty.Access.WRITE_ONLY)
```

Comment:
If the annotation is Immutable (line 12), why the access is WRITE_ONLY?

Reply:
This is auto-generated code and this was added because we removed all the property setters from this class